reactioncommerce/reaction
# 2332 Fix Improper Cart Merge

Resolves #2332 

**Testing**
- Clone this Branch and Log In
- Add `Red Option` of `Basic Reaction Product` to Cart
- Log out and add `Red Option` of `Basic Reaction Product` to Cart again
- Log In again and observe that the quantity of `Red Option` has been increased rather than having a new `Red Option` item
